crates/rpc/rpc-types-compat/src/transaction/mod.rs

```diff
@@ -33,7 +34,7 @@ pub fn from_recovered<T: TransactionCompat>(
     tx: TransactionSignedEcRecovered,
     resp_builder: &T,
 ) -> T::Transaction {
-    resp_builder.fill(tx, TransactionInfo::default())
+    resp_builder.fill(tx, TransactionInfo::default()).unwrap()
```

Comment:
```suggestion
    resp_builder.fill(tx, TransactionInfo::default())
```
same, we don't want to panic here on error